Resources/views/templates/CommonAdmin/ListTemplate/thead.php.twig

```diff
@@ -42,7 +42,7 @@
                 <a href="{{ echo_twig("path(listRoute, { 'sort': '" ~ column.sortOn ~ "', 'order_by': 'asc'})") }}">
             {{ echo_endif () }}
         {%- endif -%}
-        {{ echo_twig("'" ~ column.label|addslashes ~ "'|trans({}, '" ~ i18n_catalog|default("Admin") ~ "')") }}
+        {{ echo_twig("'" ~ column.label|addslashes ~ "'|trans({}, '" ~ i18n_catalog|default("Admin") ~ "')") }}{{ echo_if (column.isSortable) }}<i class="sort fa-solid fa-fw fa-sort{{ echo_if ("sortColumn == '" ~ column.sortOn ~ "'") }}{{ echo_if ("sortOrder == 'ASC'") }}-up{{ echo_else () }}-down{{ echo_endif () }}{{ echo_else () }} transparent{{ echo_endif () }}"></i>{{ echo_endif () }}
```

Comment:
Instead of this whole if, both icons can be rendered with the a marker class for up/down. This way we can keep basically all existing CSS rules, and with a small wrapper combined with flex-reverse we can even very easily keep ltr/rtl support.